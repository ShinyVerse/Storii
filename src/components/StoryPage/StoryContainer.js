import React from 'react';

export default function StoryContainer() {
  return (
    <ul id='story'></ul>
  )
}
