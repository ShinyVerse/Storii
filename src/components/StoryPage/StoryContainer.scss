$scrollBGColor: white;

#story {
  height: 70vh;
  width: calc(75%);
  margin-left: 3%;
  overflow: hidden;
  overflow-y: scroll;
  display: flex;
  flex-direction: column;
  align-self: center;
}

#story::-webkit-scrollbar-track
{
	-webkit-box-shadow: inset 0 0 3px rgba(0,0,0,0.3);
	border-radius: 10px;
	background-color: $scrollBGColor;
}

#story::-webkit-scrollbar
{
	width: 12px;
	background-color: $scrollBGColor;
}

#story::-webkit-scrollbar-thumb {
  border-radius: 10px;
   -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.5);
   background-color: #0489B1;
}
