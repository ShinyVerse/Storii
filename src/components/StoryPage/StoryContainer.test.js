import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import StoryContainer from './StoryContainer';

describe('StoryContainer', () => {
  test('renders a ul with id story to the page', () => {
    const wrapper = shallow(<StoryContainer />);

    expect(wrapper.find('#story').length).toBe(1);
  })

  test('snapshot for StoryContainer', () => {
    const tree = renderer
        .create(<StoryContainer />)
        .toJSON();
    expect(tree).toMatchSnapshot();
  });
})
