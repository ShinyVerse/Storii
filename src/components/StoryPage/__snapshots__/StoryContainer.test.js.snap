// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryContainer snapshot for StoryContainer 1`] = `
<ul
  id="story"
/>
`;
